examples/funloc/analysis_fun.py

```diff
@@ -98,10 +98,14 @@
 params.compute_rank = True  # compute rank of the noise covariance matrix
 params.cov_rank = None  # preserve cov rank when using advanced estimators
 
-# Define number of SSP projectors. Columns correspond to Grad/Mag/EEG chans
-params.proj_nums = [[1, 1, 0],  # ECG
-                    [1, 1, 2],  # EOG
-                    [0, 0, 0]]  # Continuous (from ERM)
+# Define number of SSP projectors.
+# Three lists, one for ECG/EOG/continuous, each list with entries for
+# Grad/Mag/EEG. Can also be a per-subject dict (or defaultdict), like:
+# Can also be a dict (including defaultdict). So let's set some defaults:
+params.proj_nums = dict(
+    subj_01=[[3, 3, 0], [3, 3, 2], [0, 0, 0]],
+    subj_02=[[3, 3, 0], [3, 3, 2], [0, 0, 0]],
+    )
```

Comment:
It might be possible, but I haven't seen anyone do it. The projectors do have an "explained variance", though, so we could pretty easily add support for `float` like
```
proj_nums = [0.95, 0.95, 0.]
```
meaning, take however many ECG, EOG, and empty-room projectors for each subject that you need to account for at least 95, 95, and 0% of the variance of the artifact evoked / epochs object.

Reply:
Yes, I think that would be a great idea! I'd even advocate variance explained as the default.